src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is working right now!");
```

Comment:
The message printed should be more informative and indicate the type of machine that is working. Consider including the class name in the message.